src/dxtbx/format/FormatCBFMiniEigerChessID7B2.py

```diff
@@ -0,0 +1,39 @@
+"""
+An implementation of the FormatCBFMiniEiger image reader for the Eiger2 16M
+detector at the CHESS ID7B2 beamline, which has an inverted goniometer axis.
+"""
+
+from __future__ import annotations
+
+import sys
+
+from dxtbx.format.FormatCBFMiniEiger import FormatCBFMiniEiger
+
+
+class FormatCBFMiniEigerChessID7B2(FormatCBFMiniEiger):
+    """A class for reading mini CBF format Eiger16M images for S/N E-32-0123
+    installed at CHESS ID7B2, which has an inverted goniometer axis."""
+
+    @staticmethod
+    def understand(image_file):
+        """Check to see if this looks like an Eiger mini CBF format image,
+        i.e. we can make sense of it."""
+
+        header = FormatCBFMiniEiger.get_cbf_header(image_file)
+        for record in header.split("\n"):
+            if "Detector: Dectris EIGER2 Si 16M, S/N E-32-0123" in record:
```

Comment:
Suggest 

```
if "# Detector: ...
```

in record to make it clear to the reader that we are looking in the `#` demarked metadata island in the header